app/_src/kubernetes-ingress-controller/concepts/expression-based-router.md

```diff
@@ -0,0 +1,91 @@
+
+### Instructions
+
+For {{site.kic_product_name}}, you must enable `ExpressionRoutes` and `CombinedRoutes` (enabled by default) so {{site.kic_product_name}} can translate Kubernetes resources to expression-based
+{{site.base_gateway}} routes. For {{site.base_gateway}}, you must configure `KONG_ROUTER_FLAVOR` for `expressions` to use expression-based router.
+
+You can use the following `values.yaml` to install {{site.kic_product_name}} using the expression-based router by helm:
+
+```yaml
+image:
+  repository: kong
+  tag: "3.3" # 3.0 or later
+
+env:
+  database: "off"
+  router_flavor: "expressions" # sets KONG_ROUTER_FLAVOR to "expressions"
+
+ingressController:
+  enabled: true
+  image:
+    repository: kong/kubernetes-ingress-controller
+    tag: "2.10" # 2.10 or later
+  env:
+    feature_gates: "ExpressionRoutes=true" # enables the "ExpressionRoutes" feature gate
+```
+
+Then, use Helm to install {{site.base_gateway}}:
+
+```bash
+  helm upgrade --install controller kong/kong -n kong --create-namespace -f values.yaml
+```
+
+## Kubernetes resource support
+
+The following table displays which Kubernetes resources are supported when translating to expression-based routes is enabled in {{site.kic_product_name}}:
+
+| Kubernetes resource | Supported? |
+| ------------------- | ---------- |
+| [`Ingress`](ingress) | ✅ &nbsp; |
+| [`HTTPRoute`](gateway-api-httproute) | ✅ &nbsp; |
+| [`GRPCRoute` in gateway APIs](gateway-api-grpcroute) | ✅ &nbsp; |
+| [`knative.Service`](knative-service) | ❌ &nbsp; |
+| [`TCPIngress`](crd-tcpingress) | ❌ &nbsp; |
+| [`UDPIngress`](crd-udpingress) | ❌ &nbsp; |
+| [`TCPRoute`](gateway-api-tcproute) | ❌ &nbsp; |
+| [`TLSRoute`](gateway-api-tlsroute) | ❌ &nbsp; |
+| [`UDPRoute`](gateway-api-udproute) | ❌ &nbsp; |
+
+## Unsupport methods of overriding routes
+
+Because Kubernetes objects can't fully express {{site.base_gateway}} configuration specifications, {{site.kic_product_name}} provides methods to override services, routes, plugins, consumers, and other objects in {{site.base_gateway}} configurations after translating from Kubernetes objects. These methods include overriding using the `KongIngress` resource and annotations
+of resources. When the expression-based router is enabled, some of the methods of overriding routes in {{site.base_gateway}} configurations after translating are not supported:
+
+* You can't override routes in {{site.base_gateway}} configurations using `KongIngress` isn't supported when expression routes is enabled. The `route` fields of `KongIngress` resources will be ignored when {{site.kic_product_name}} translate Kubernetes resources to expression based routes.
+* You can't override `Ingress` with the `konghq.com/path_handling` and `konghq.com/regex_priority` annotations because the `path_handling` and `regex_priority` fields are not supported in {{site.base_gateway}} route configurations when {{site.base_gateway}} is running expression-based router.
```

Comment:
```suggestion
* You can't override `Ingress` with the `konghq.com/path_handling` and `konghq.com/regex_priority` annotations because the `path_handling` and `regex_priority` fields are not supported in {{site.base_gateway}} route configurations when {{site.base_gateway}} is running the `expressions` router.
```